<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>장비 설정</string>
  </property>
  <widget class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="plc_tab">
      <attribute name="title">
       <string>PLC 설정</string>
      </attribute>
      <widget class="QFormLayout" name="plc_formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="plc_port_label">
         <property name="text">
          <string>포트:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="plc_port_combo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="plc_baudrate_label">
         <property name="text">
          <string>통신속도:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="plc_baudrate_combo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="plc_parity_label">
         <property name="text">
          <string>패리티:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="plc_parity_combo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>N (None)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E (Even)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>O (Odd)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="plc_stopbits_label">
         <property name="text">
          <string>정지비트:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="plc_stopbits_combo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="plc_timeout_label">
         <property name="text">
          <string>타임아웃(초):</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="plc_timeout_spin">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="plc_test_button">
         <property name="text">
          <string>연결 테스트</string>
         </property>
        </widget>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="scanner_tab">
      <attribute name="title">
       <string>바코드 스캐너</string>
      </attribute>
      <widget class="QFormLayout" name="scanner_formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="scanner_port_label">
         <property name="text">
          <string>포트:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="scanner_port_combo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="scanner_baudrate_label">
         <property name="text">
          <string>통신속도:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="scanner_baudrate_combo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="scanner_timeout_label">
         <property name="text">
          <string>타임아웃(초):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="scanner_timeout_spin">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="scanner_test_button">
         <property name="text">
          <string>연결 테스트</string>
         </property>
        </widget>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="printer_tab">
      <attribute name="title">
       <string>바코드 프린터</string>
      </attribute>
      <widget class="QFormLayout" name="printer_formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="printer_port_label">
         <property name="text">
          <string>포트:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="printer_port_combo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="printer_baudrate_label">
         <property name="text">
          <string>통신속도:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="printer_baudrate_combo">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="printer_timeout_label">
         <property name="text">
          <string>타임아웃(초):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="printer_timeout_spin">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="printer_test_button">
         <property name="text">
          <string>연결 테스트</string>
         </property>
        </widget>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QHBoxLayout" name="button_layout">
     <item>
      <widget class="QPushButton" name="refresh_ports_button">
       <property name="text">
        <string>포트 새로고침</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
    </widget>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
